### PR TITLE
chore(windows): remove WM_UNICHAR completely

### PR DESCRIPTION
**Pull request by rc-swag**
Fixes: #8677 

For this PR we can just load two different keyman keyboards and test that characters are still being output

## User Testing

# TEST_TAMIL_OUTPUT

Install Tamil 99 keyboard
*  Open Notepad, select the Tamil Keyboard 
* Type in some characters and ensure they are output
* Open MS `Word` if you have it otherwise, Firefox
* Verify that typing produces the expected output

# TEST_KHMER_ANGKOR

Install Khmer Angkor keyboard

*  Open Notepad, select the Khmer_Angkor Keyboard 
* Type in some characters and ensure they are output
* Open MS `Word` if you have it otherwise, Firefox
* Verify that typing produces the expected output

### CODE REVIEW

File: windows/src/engine/keyman32/appint/aiWin2000Unicode.cpp [hunk 1]
```diff
@@ -45,11 +45,6 @@
 
 #define KEYMAN_MOREPOST	"WM_KMMOREPOST"
```
Comment (mcdurdin) [inline windows/src/engine/keyman32/appint/aiWin2000Unicode.cpp:46]: delete this too

File: windows/src/engine/keyman32/k32_dbg.cpp [hunk 2]
```diff
@@ -194,7 +194,6 @@ char *msgnames[] = {
 "WM_SYSCHAR",
 "WM_SYSDEADCHAR",
 "WM_x108",
-"WM_UNICHAR"
```
Comment (mcdurdin) [inline windows/src/engine/keyman32/k32_dbg.cpp:197]: don't delete this :)

File: windows/src/engine/keyman32/k32_dbg.cpp [hunk 3]
```diff
@@ -218,16 +217,6 @@ void DebugMessage(LPMSG msg, WPARAM wParam)  // I2908
...
-      (unsigned int) GetMessageExtraInfo());
-	else if(msg->message >= WM_KEYDOWN && msg->message <= WM_UNICHAR)
-    wsprintf(ds, "DebugMessage(%x, %s, wParam: '%c' (U+%04X), lParam: %X) [message flags: %x time: %d extra: %x]",
-      PtrToInt(msg->hwnd),
-      msgnames[msg->message-WM_KEYDOWN],
-      msg->wParam,
-      msg->wParam,
-      (unsigned int) msg->lParam,
-      wParam,
-      (int) msg->time,
```
Comment (mcdurdin) [inline windows/src/engine/keyman32/k32_dbg.cpp:230]: leave this as is, don't delete

File: windows/src/test/manual-tests/msghooklister/src/App/wm_messages.h [hunk 4]
```diff
@@ -266,7 +266,6 @@ static const LPCTSTR WM_MESSAGE_STRINGS[] = {
 	TEXT("WM_SYSCHAR"),
 	TEXT("WM_SYSDEADCHAR"),
 	TEXT("WM_YOMICHAR"),
-	TEXT("WM_UNICHAR"),
```
Comment (mcdurdin) [inline windows/src/test/manual-tests/msghooklister/src/App/wm_messages.h:269]: leave this -- it's indexed to message ids

**Review by mcdurdin — COMMENTED** [review]
very quick comments from our paired review

File: windows/src/engine/keyman32/k32_dbg.cpp (hunk 3 above)
Comment (mcdurdin) [reply windows/src/engine/keyman32/k32_dbg.cpp:230]: this code hasn't yet been restored

**Review by mcdurdin — CHANGES_REQUESTED** [review]

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_TAMIL_OUTPUT (PASSED):** Tested with the attached PR build (Keyman 17.0.87-alpha-test-8744) in Windows 11 OS (VM) and here is my observation: 1. Installed Tamil 99 keyboard. 2. Able to use Tamil keyboard in the Notepad app as well as in the Firefox browser. Thanks, Ross.

..in NotePad



..in Firefox browser



* **TEST_KHMER_ANGKOR (PASSED):** 1. Installed Khmer Angkor keybord. 2. Able to use the Khmer Angkor keyboard to type in khmer in the Notepad app as well as in the Firefox browser. Seems to be working fine. Thanks. 

..in Notepad




.. Firefox browser

**Pull request merged**